air-script/tests/aux_trace/aux_trace.masm

```diff
@@ -45,6 +62,16 @@ proc.compute_integrity_constraint_divisor
     # => [divisor_1, divisor_0, ...]
 end # END PROC compute_integrity_constraint_divisor
 
+# Procedure to evaluate each integrity constraints.
```

Comment:
minor nit: should this be "Procedure to evaluate numerators of all integrity constraints."?

Reply:
changed